src/index.js

```diff
@@ -7,6 +7,9 @@ import {css, styleSheet} from 'glamor'
 import shouldForwardProperty from './should-forward-property'
 
 const htmlTagNames = require('html-tag-names')
+const svgTagNames = require('svg-tag-names')
+
+const domElements = htmlTagNames.concat(svgTagNames)
```

Comment:
Do we need to deduplicate this array? `svg` and `iframe` are repeated, but there may be others.